src/components/Person/Person.jsx

```diff
@@ -1 +1,23 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+
+    {person.age && (
+      <p className="Person__age">
+        {`I am ${person.age}`}
+      </p>
+    )
+    }
+
+    <p className="Person__partner">
+      {person.isMarried ? (
+        `${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}`
+      ) : (
+        'I am not married'
+      )
+      }
+    </p>
```

Comment:
do not make nested ternary operators, move one condition into fn or variable

Reply:
<img width="581" alt="image" src="https://github.com/mate-academy/react_person/assets/70220667/b5708c1f-03ec-4c76-8430-8c3722a59ad0">
still not fixed


decide before jsx what text will be rendered and just render in jsx.